tests/sqlparser_postgres.rs

```diff
@@ -2261,6 +2262,51 @@ fn parse_create_index() {
             assert!(if_not_exists);
             assert_eq_vec(&["col1", "col2"], &columns);
             assert!(include.is_empty());
+            assert!(with.is_empty());
+        }
+        _ => unreachable!(),
+    }
+}
+
+#[test]
+fn test_create_index_with_with_clause() {
```

Comment:
since the functionality uses a method on the dialect, we can rewrite the test in [this style](https://github.com/sqlparser-rs/sqlparser-rs/blob/222b7d127acb265472cc9bf2e81ef2453a53a9ce/tests/sqlparser_common.rs#L2765) in common.rs so that future dialects that gain support get test coverage automatically

Reply:
Cool